crates/stackable-operator/src/kvp/key.rs

```diff
@@ -109,6 +109,12 @@ impl Display for Key {
     }
 }
 
+impl From<&Key> for String {
+    fn from(value: &Key) -> Self {
+        value.to_string()
+    }
+}
```

Comment:
It makes using it in a SNAFU error slightly nicer (`.context(BlahSnafu { key })` instead of `.with_context(|_| BlahSnafu { key: key.to_string() })`), but.. arguably the SNAFU error should just take a `Key` instead to begin with.

Reply:
1e4cffe158ee825f332b5f5af2a29f4ec912857c